release/gittools.py

```diff
@@ -53,7 +53,7 @@ def delete_tag(self, repo, tag):
         :param str tag: The tag to delete (e.g. 'v1.0.0')
         """
         logging.info('Deleting tag %(tag)s in repository: %(repo)s', dict(tag=tag, repo=repo))
-        ref_url = "%s/%s/git/refs/tags/%s" % (self.GIT_API_REPOS, repo, tag)
+        ref_url = "%s/%s/tags/%s" % (self.GIT_API_REPOS, repo, tag)
```

Comment:
Sure this is the url?

Reply:
I was also surprised,
but this works and the old isn't working.